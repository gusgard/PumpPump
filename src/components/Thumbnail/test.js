import { shallow } from 'enzyme'
import React from 'react'

import { Thumbnail } from '@components'

it('renders correctly', () => {
  const wrapper = shallow(<Thumbnail />)
  expect(wrapper).toMatchSnapshot()
})
