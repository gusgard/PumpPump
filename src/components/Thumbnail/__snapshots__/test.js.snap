// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Image
    onLoadEnd={[Function]}
    onLoadStart={[Function]}
    resizeMode="cover"
    source={
      Object {
        "testUri": "../../../src/images/placeholderUser/user.png",
      }
    }
    style={
      Object {
        "borderRadius": 50,
        "height": 100,
        "width": 100,
      }
    }
  />
</View>
`;
